php/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.0-fpm-alpine
+FROM php:8.1-fpm-alpine
```

Comment:
```suggestion
FROM php:8.1.0-fpm-alpine
```

Reply:
not sure about that. after digging on docker hub, I find that the above tag was published after this `PR` was created

https://hub.docker.com/_/php?tab=tags&page=1&name=8.1-alpine 

not sure what triggers this `PR` tho